packages/block-library/src/comment-template/edit.js

```diff
@@ -45,18 +131,21 @@ export default function CommentTemplateEdit( {
 				comments: getEntityRecords( 'root', 'comment', {
```

Comment:
Unrelated to this PR, but something we should clarify later is whether we should limit the fields that we fetch for the comments:

<img width="1009" alt="Screenshot 2021-12-02 at 12 11 07" src="https://user-images.githubusercontent.com/699132/144411458-bcd89843-7847-4d27-b270-b531ffeae907.png">

Maybe we should use the `view` context since we are not going to allow editing comments in this place? See more at:
https://developer.wordpress.org/rest-api/reference/comments/#schema

/cc @c4rl0sbr4v0

Reply:
Yep. I think it would be a quick win, that way we are not exposing data that is more related to other contexts like IP, User-Agent, etc.

Reply:
Oh, I didn't know about the `contexts` - gonna take a look although it seems like [`view` is already the default](https://developer.wordpress.org/rest-api/reference/comments/#:~:text=Arguments-,context,-Scope%20under%20which) 🤔

Reply:
I could use the `embed` context here because it includes fewer fields. We actually only need the `id` and `parent` fields.

Reply:
It seems to work just fine when using the `embed` context, although I notice that we load all the individual comments are loaded with context `edit` anyway when a post is loaded:

<img width="776" alt="Screen Shot 2021-12-03 at 17 29 55" src="https://user-images.githubusercontent.com/5417266/144684184-ee213b9a-cef0-4358-9745-e852541d0456.png">

so any wins here are likely pretty small 🙂

Reply:
Anyway, I've added `context: 'embed'` in 71c7d000481b9e41633f8faf9d63a3b83c84a29d

Reply:
I don't see any place where we would use `embed` as context, but `view` is quite popular. We can align that later.